src/app/zap-templates/zcl/data-model/chip/thermostat-cluster.xml

```diff
@@ -660,6 +711,32 @@ limitations under the License.
       </mandatoryConform>
     </command>
 
+    <command code="0x07" source="client" name="AddThermostatSuggestion" response="AddThermostatSuggestionResponse">
```

Comment:
apiMaturity on all the commands, please.

Reply:
done.